docs/book/src/migration/project/v2_v3.md

```diff
@@ -68,47 +68,21 @@ fine.
 
 ## Migrating your projects to use v3+ plugins
 
-<aside class="note warning">
-
-<h1>Note</h1>
-
-`v3+` plugins are still in alpha phase and are not scaffolded by default. You are able to use them, try it out and get all improvements made so far by initializing projects with the arg `--plugins=go.kubebuilder.io/v3-alpha`:
-
-```sh
-kubebuilder init --domain my.domain --plugins=go.kubebuilder.io/v3-alpha
-```
-
-</aside>
-
-<aside class="note warning">
-
-<h1>Note</h1>
-
-Currently, the plugin `v3-alpha` has NO breaking changes. However, until it is declared stable breaking changes can be made related to K8s API deprecations of `v1beta1` versions of `CustomResourceDefinition` and `ValidatingWebhookConfiguration` and to upgrade the `cert-manager`. More info:
-
-- [cert-manager related configuration should be migrated to cert-manager.io/v1 #1666
-](https://github.com/kubernetes-sigs/kubebuilder/issues/1666)
-- [Set preserveUnknownFields to false in the CRD conversion webhook patch #933](https://github.com/kubernetes-sigs/kubebuilder/issues/933)
-- [Migrate existing KB project to v1 CRDs and Webhooks with minimal user effort #1065
-](https://github.com/kubernetes-sigs/kubebuilder/issues/1065)
-
-</aside>
-
```

Comment:
We can add here //TODO and cove that in the follow up

Reply:
Not sure what you mean here. I removed those two notes because now that go/v3 is the default plugin, they no longer apply.

Reply:
It is in the migration doc. After we merge that I will update the PR https://github.com/kubernetes-sigs/kubebuilder/pull/1916 where we will cover it. so, for now, a //TODO shows fine..